src/pages/LearningResources.js

```diff
@@ -1,231 +1,178 @@
-              target='_blank'
-              rel='noopener noreferrer'
-            >
-              Team Treehouse
-            </a>
-          </li>
-
-          <li className='underline mb-3 text-blue-700 hover:text-teal-600'>
-            <a
-              href='https://adamlearns.com/'
-              target='_blank'
-              rel='noopener noreferrer'
-            >
-              Adam Learns
-            </a>
-          </li>
-
-          <li className='underline mb-3 text-blue-700 hover:text-teal-600'>
-            <a
-              href='https://udemy.com/'
-              target='_blank'
-              rel='noopener noreferrer'
-            >
-              Udemy
-            </a>
-          </li>
-
-          <li className='underline mb-3 text-blue-700 hover:text-teal-600'>
-            <a
-              href='https://frontendmasters.com/'
-              target='_blank'
-              rel='noopener noreferrer'
-            >
-              Frontend Masters
-            </a>
-          </li>
-        </motion.ul>
+      <div className='container justify-start my-4 px-8 border-b-1'>
+        <h3 className='text-xl text-blue-900 sm:text-2xl md:text-3xl'>Free Resources</h3>
       </div>
-    </motion.div>
+
+      <div className='content-center flex grid-flow-col flex-wrap'>
+        <ResourceCard
+          title='FreeCodeCamp'
+          resourceimg='freecodecamp.jpg'
+          tag='free'
+          description='A nonprofit community that helps you learn to code by building projects. Learn to code by completing coding challenges and building projects. Earn verified certifications along the way.'
+          resourceurl='freecodecamp.org'
+        />
+        <ResourceCard
+          title='The Beginner&#39;s Guide to React'
+          resourceimg='beg-guide-react.jpg'
+          tag='free'
+          description='This course is for React newbies and anyone looking to build a solid foundation. It’s designed to teach you everything you need to start building web applications in React right away.'
+          resourceurl='egghead.io/courses/the-beginner-s-guide-to-react'
+        />
+
+        <ResourceCard
+          title='Devtips'
+          resourceimg='devtips.jpg'
+          tag='free'
+          description='DevTips is a weekly show for YOU who want to be inspired and learn about programming. Hosted by David and MPJ - two notorious bug generators and teachers.'
+          resourceurl='youtube.com/c/DevTipsForDesigners'
+        />
+
+        <ResourceCard
+          title='Quentin Watt'
+          resourceimg='quentin.jpg'
+          tag='free'
+          description='Learn how to code JavaScript, and PHP using popular frameworks like Vue.js and Laravel. Aimed at equipping you with the skills you need to develop your career. '
+          resourceurl='youtube.com/user/QuentinWatt'
+        />
+
+        <ResourceCard
+          title='Wes Bos Youtube'
+          resourceimg='wesbos.jpg'
```

Comment:
This should be `resourceimg='wesbos-youtube.jpg'

Reply:
Swapped Wes' image links